src/main/content/_assets/css/guide.scss

```diff
@@ -212,11 +214,11 @@ header {
 }
 #guide-content .code_command > .content:before {
     top: 0px;
-    background: linear-gradient(to top right, #F1F4FE 48%, #c8d6fb calc(48% + 1px), transparent calc(48% + 2px));
+    background: linear-gradient(to top right, #f4f5f7 48%, $color-kabanero calc(48% + 1px), transparent calc(48% + 2px));
 }
 #guide-content .code_command > .content:after {
     bottom: 0px;
-    background: linear-gradient(to bottom right, #F1F4FE 48%, #c8d6fb calc(48% + 1px), transparent calc(48% + 2px));
+    background: linear-gradient(to bottom right, #f4f5f7 48%, $color-kabanero calc(48% + 1px), transparent calc(48% + 2px));
 }
 /* End of pointed div */
 
```

Comment:
The guide content section headers could look a little more "Kabanero-colored". On line 244, in the "#guide-content .sect h2" rule, try changing background to #C43804AA.

I would suggest the same color change at the end-of-guide section.  That color is set in end-of-guide.css in the "#end_of_guide h2" box-shadow rule.